hooks/viewer/useViewer.js

```diff
@@ -25,7 +28,27 @@ export default function useViewer() {
   }, [accessToken, viewer]);
 
   useEffect(() => {
-    if (viewer) setAccount(viewer);
+    if (viewer) {
+      setAccount(viewer);
+
+      // identify the user
+      const isAuthenticated = viewer?._id;
+      const isTracked = localStorage.getItem("isTracked");
```

Comment:
Is this really necessary?

If `viewer` changes, I'd guess it's okay to send another identify call / prefer risking duplicate calls over outdated data - If any provider is not designed to deal with this for some reason, I guess some identify logic could be applied on the provider level to hash values and avoid duplicates.

Right now, if I log out and next log in in with another account, the new user would not be identified with this in place if I get it correctly.